File: auto-docs/executables/python/percent_error_bar.py
import plotly.plotly as py
from plotly.graph_objs import *
py.sign_in('TestBot', 'r1neazxo9w')

data = Data([
    Scatter(
        x=[0, 1, 2],
        y=[6, 10, 2],
        error_y=ErrorY(
            type='percent',
            value=50,
            visible=True
        )
    )
])
plot_url = py.plot(data, filename='percent-error-bar', auto_open=False)

File: auto-docs/executables/nodejs/percent_error_bar.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')
var data = [
  {
    x: [0, 1, 2], 
    y: [6, 10, 2], 
    error_y: {
      type: "percent", 
      value: 50, 
      visible: true
    }, 
    type: "scatter"
  }
];
var graph_options = {auto_open: false, fileopt: "overwrite", filename: "percent-error-bar"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});